### DISCUSSION TITLE
Sample electron app without webuiextension window

### DISCUSSION THREAD

**Discussion by Jenifer-TheCoder** [samuelmaddock/electron-browser-shell#26] [Q&A]
I have created sample electron app following:https://www.electronjs.org/docs/tutorial/quick-start. My intention is to install electron & electron-chrome-extensions to see independently how this node module works with any custom electron application.

**I did the steps below:**
- Created electron app with main file as https://github.com/samuelmaddock/electron-browser-shell/blob/master/packages/shell/index.js
- Note have duplicated files of index.js of shell,main.js of shell/browser/ preload.js etc into single folder.
- Since I don't want webuiextension window,i had replaced webuiUrl variable with say "google.com" URL  in https://github.com/samuelmaddock/electron-browser-shell/blob/master/packages/shell/browser/main.js
- Noticed that the electron-chrome-extension code creates frameless window with browserview added to it.
- Question here is:
 **if i want to add "`extensions.addTab...`" code to any Electron app what and all need to be cut?**

What I did are:
1. Removed weuiextension related code from index.js(https://github.com/samuelmaddock/electron-browser-shell/blob/master/packages/shell/index.js)
2. If i get to see the DOM content of the electron app, it is still with kind of webuiextension html window with a single tab as parent to frame, just wondering anywhere else I have to act on it.

**Comment by samuelmaddock** [comment]
Usage examples are included in the readme of the electron-chrome-extensions package: https://github.com/samuelmaddock/electron-browser-shell/tree/master/packages/electron-chrome-extensions#usage

**Comment by Jenifer-TheCoder** [comment]
Update:
By loading explicitly with sample usage code,could see chrome extension been working.But extensions.addTab function adds tab type UI while adding the tab looks like.If we extract the DOM data of the HTML content there is browser like tab as in snap:

![image](https://user-images.githubusercontent.com/43472476/118667301-84fc1580-b811-11eb-9eb6-ca27209e7c1b.png)


Thanks!!
